src/views/CustomTileDialog/AddCustomTile/index.jsx

```
@@ -39,18 +40,34 @@ export default function AddCustomTile({
+      tags: [],
+    });
   }
 
-  async function submitNewTile() {
-    const { tileOption, action, tags } = formData;
+  async function submitNewTile(event) {
+    // Check if there's text in the tag input field and add it to tags
+    const tagInput = document.querySelector('input[name="tags"]');
+    let currentTags = [...formData.tags];
+    
+    if (tagInput && tagInput.value.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)